cvpods/engine/runner.py

```diff
@@ -14,6 +14,7 @@
 from cvpods.evaluation import (
     DatasetEvaluator,
     inference_on_dataset,
+    inference_on_files,
```

Comment:
you might put all modifications related to 'inference_on_files' to an individual pr.